certified-connectors/Actsoft/apiDefinition.swagger.json

```diff
@@ -0,0 +1,14408 @@
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "X-API-Version",
+            "in": "header",
+            "description": "Requested current minor version. Latest version is 9.",
+            "required": false,
+            "type": "integer",
+            "x-ms-visibility": "internal",
+            "x-ms-summary": "X-API-Version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "This is returned when there are no records."
+          },
+          "401": {
+            "description": "Request was not applied, because user credentials could not be authenticated."
+          },
+          "500": {
+            "description": "This error is always an indication of a bug on the server side; possibly due to an unhandled exception."
+          },
+          "503": {
+            "description": "Service temporarily unavailable because other services (such as Azure service, or a required 3rd party service like Google API) are unavailable."
+          }
+        }
+      }
+    },
+    "/forms/v1/poll": {
+      "get": {
+        "tags": [
+          "Forms API"
+        ],
+        "summary": "Form data poll",
+        "description": "The Forms API polling endpoint allows your application to continually poll and process recently submitted standalone forms.\r\n\r\nIntegration:\r\nThis endpoint is designed to work out of the box with Microsoft Flow as a \u201cpolling trigger\u201d integration. Alternatively, you may develop your own \r\ncustom integration using any language.\r\n\r\nUsage:\r\nIn a typical scenario, the first request to this endpoint should not provide any parameters. There are two (2) possible successful responses:\r\n* HTTP Response code 200 - Recently submitted form records exist; response body will display form data.\r\n* HTTP Response code 202 \u2013 No recently submitted records exist; response body will be empty.\r\n\r\nIn both cases, there are two (2) important headers in the response:\r\n* Location \u2013 The URL for the next request.\r\n* Retry-After \u2013 The recommended time interval in seconds for next request. For example, if there is data to retrieve, the Retry-After header will have a value of 1 or 60. \r\nIf there is no data to retrieve, the value of the Retry-After header will be 300.\r\n\r\nNote: These are just recommended values \u2013 your application may choose to poll at a different frequency depending on your requirements. After the first request, all subsequent \r\nrequests should use the URL provided in the Location header of the most recent response. Do not add more parameters to the URL, as it already contains the necessary \r\nparameters required to ensure forms are not missing from the request.\r\n\r\nResults are automatically sorted by ModifiedUtc and then by FormHeaderId, in ascending order. Sort parameters are not configurable.\r\nModifiedUtc definition: internal time tag = time of latest form data change.",
```

Comment:
@rdameron Your swagger has unicodes which we do not support at the moment, Can you please replace them with English language characters. Search  with "\u" in the swagger to see them

Reply:
Ok. Thanks for the search tip. Unicodes removed.